docs/pages/docs/best-practices.mdx

```diff
@@ -36,6 +36,9 @@ Our documentation is primarily built with [Next](https://github.com/gravitationa
 **Diagrams:**
 
 - Use [Teleport's lucidchart library](https://app.lucidchart.com/lucidchart/dfcf1f4a-5cf0-4758-8ebb-f6ea86900aba/edit) to create diagrams with a consistent design language.
```

Comment:
Changes in here are backported for https://github.com/gravitational/teleport/pull/6809